drivers/gpio/gpio_tle9104.c

```diff
@@ -433,16 +433,18 @@ static int tle9104_init(const struct device *dev)
 		}
 	}
 
-	result = gpio_pin_configure_dt(&config->gpio_reset, GPIO_OUTPUT_ACTIVE);
-	if (result != 0) {
-		LOG_ERR("failed to initialize GPIO for reset");
-		return result;
-	}
+	if (config->gpio_reset.port != NULL) {
+		result = gpio_pin_configure_dt(&config->gpio_reset, GPIO_OUTPUT_ACTIVE);
```

Comment:
Looks like this is missing a call to `gpio_is_ready_dt()`?

Reply:
Good point, fixed it.